### PR TITLE
Added basic setup for Angular

### PR DESCRIPTION
**Pull request by aka-jain**
@deshraj Added basic setup for EvalAI frontend.
cc @spyshiv @RishabhJain2018

### CODE REVIEW

**Comment by spyshiv** [comment]
@aka-jain We should setup project in root directory. Creating `evalai` folder inside root directory will cause additional step (i.e. `cd evalai`) in development

**Comment by aka-jain** [comment]
Angular made this directory but it's own. Basically ng new app generate a new dir and setup all the files in it. 

But we can check how to overcome this problem.

**Comment by spyshiv** [comment]
@aka-jain : Yes. we can overcome this. Check this out.
https://github.com/angular/angular-cli/wiki/new#options
I'm on it.

**Comment by spyshiv** [comment]
@aka-jain : I have updated the basic setup. Everything is working fine including unit test and e2e test. Please check changes once so that I can merge this PR.
PS: I am using angular-cli 1.6.5

File: src/app/app.component.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+h1 {
+	text-align: center;
+}
```
Comment (aka-jain) [inline src/app/app.component.css]: New line

**Comment by spyshiv** [comment]
Merging this PR

**Pull request merged**